src/frameworks/core_graphics/cg_color_space.rs

```diff
@@ -48,6 +48,19 @@ pub fn CGColorSpaceCreateWithName(env: &mut Environment, name: CFStringRef) -> C
     )
 }
 
+pub fn CGColorSpaceCreateDeviceRGB(env: &mut Environment) -> CGColorSpaceRef {
+    let isa = env
+        .objc
+        .get_known_class("_touchHLE_CGColorSpace", &mut env.mem);
+    env.objc.alloc_object(
+        isa,
+        Box::new(CGColorSpaceHostObject {
+            name: kCGColorSpaceGenericRGB,
```

Comment:
I was curious what this actually does, so I tested it on macOS:

```objc
#import "Foundation/Foundation.h"
#import "CoreGraphics/CoreGraphics.h"

int main(void)
{
    CGColorSpaceRef cs = CGColorSpaceCreateDeviceRGB();
    NSLog(@"%@", CGColorSpaceCopyName(cs));
}
```

Looks like the result is some undocumented constant:

```
$ ./CGColorSpaceCreateDeviceRGB 
2023-02-10 14:03:28.464 CGColorSpaceCreateDeviceRGB[10366:143295] kCGColorSpaceDeviceRGB
```

I was hoping it might turn out to be an alias for generic RGB depending on the system, but I guess not. Btw, that constant doesn't seem to be in the API, so I don't think it's something we're missing.

Maybe add a comment like `// TODO: figure out what characteristic this actually has on an iPhone`, because I assume we don't know if this is e.g. sRGB versus Linear RGB.

Reply:
I added a comment